### DISCUSSION TITLE
[Feature] german translation for the mail

### DISCUSSION THREAD

**Discussion by MarJun1988** [immich-app/immich#12513] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

**_German:_**
Ich würde gerne die Möglichkeit haben, die Sprache für den E-Mail-Versand auszuwählen. Besonders hilfreich wäre dies für den automatischen Versand von erstellten Alben oder bei der Anlage neuer Nutzer.

**_English:_**
I would like to have the option to select the language for email dispatch, especially for the automatic sending of created albums or when setting up new users.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9920
